eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -571,11 +583,14 @@ public function locationCopied($locationId, $newLocationId, $newParentId)
      */
     protected function getRealAliasId($locationId)
     {
-        $data = $this->gateway->loadAutogeneratedEntry('eznode:' . $locationId);
-
-        if (empty($data) || $data['id'] != 0 && $this->gateway->isRootEntry($data['id'])) {
+        // This is intentionally hardcoded and not exposed as:
+        // 1. we don't implement this configuration option
+        // 2. such option should have no place here
+        // This solution is temporary until we can break data compatibility with Legacy Stack.
+        if ($locationId == 1 || $locationId == 2) {
```

Comment:
Same as above: at least constants, even if it's only for readability. It will also let you move those very informative comments as the constant's doc :-)